src/env/env_utils.c

```diff
@@ -0,0 +1,72 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env_utils.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: alopez-g <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/10/15 21:14:27 by alopez-g          #+#    #+#             */
+/*   Updated: 2022/10/15 22:14:30 by alopez-g         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "env.h"
+
+char	*get_key_value_pair(char *key, char *value)
+{
+	char	*pair;
+	char	*tmp;
+
+	tmp = ft_strjoin(key, "=");
+	pair = ft_strjoin(tmp, value);
+	free(tmp);
+	return (pair);
+}
+
+t_error_code	new_var(char ***env, char *key, char *value)
+{
+	if (!*env)
+	{
+		*env = (char **)malloc(sizeof(char *) + 1);
+		*(*env + 1) = NULL;
+	}
+	return (tab_add(&(*env), get_key_value_pair(key, value)));
+}
+
+int	env_size(char **envp)
```

Comment:
maybe you can use tab_count() 
i think i made a func like this

Reply:
I didnt use it because it was static, i have changed it and used it